iamNarcisse/gitpay
# Task fix nan

## Description

> Fixed NaN when the suggested value is empty after been selected.

## Changes

- Task component

## Issue

> NaN displays when the suggested value is empty after being selected.

## Impacted Area

> Task page

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

-Create / log user
-Import an issue
-Create another user
-Apply to solve that issue
-Try to suggest a price and before confirm, make it empty again

## Before
> Screenshot from the state before

![](https://user-images.githubusercontent.com/43097772/80646456-a81c2180-8a64-11ea-86b8-72810b35b5ea.png)


## After
> Screenshot from the state after your Pull Request

![](https://user-images.githubusercontent.com/43097772/80646498-c08c3c00-8a64-11ea-9b63-f92f3fbcb581.png)
